args4j/examples/SampleMain.java

```diff
@@ -49,40 +49,25 @@ public static void main(String[] args) throws IOException {
     }
 
     public void doMain(String[] args) throws IOException {
-        CmdLineParser parser = new CmdLineParser(this);
-        
+
         // if you have a wider console, you could increase the value;
         // here 80 is also the default
-        parser.setUsageWidth(80);
+        ParserProperties properties = ParserProperties.defaults().withUsageWidth(80);
+        CmdLineParser parser = new CmdLineParser(this, properties);
 
         try {
-            // parse the arguments.
             parser.parseArgument(args);
 
             // you can parse additional arguments if you want.
             // parser.parseArgument("more","args");
-
-            // after parsing arguments, you should check
-            // if enough arguments are given.
-            if( arguments.isEmpty() )
-                throw new CmdLineException(parser,"No argument is given");
-
-        } catch( CmdLineException e ) {
-            // if there's a problem in the command line,
-            // you'll get this exception. this will report
-            // an error message.
-            System.err.println(e.getMessage());
-            System.err.println("java SampleMain [options...] arguments...");
-            // print the list of available options
-            parser.printUsage(System.err);
-            System.err.println();
-
-            // print option sample. This is useful some time
-            System.err.println("  Example: java SampleMain"+parser.printExample(ALL));
-
+        } catch (CmdLineException e) {
+            e.printStackTrace();
             return;
         }
 
+        if( arguments.isEmpty() )
+            System.out.println("No args given. Example useage: java SampleMain" + parser.printExample(OptionHandlerFilter.ALL));
```

Comment:
Removing the help message on parsing error was a mistake, thanks for correcting that.

The second change depends on whether the program should run given no arguments. However, programs that use args4j will likely require some arguments to run correctly. I agree with this change as well.